components/clientComponents/forms/Form/Form.tsx

```diff
@@ -324,10 +332,35 @@ export const Form = withFormik<FormProps, Responses>({
   validate: (values, props) => validateOnSubmit(values, props),
 
   handleSubmit: async (values, formikBag) => {
+    const getValuesForConditionalLogic = () => {
+      const inputHistoryValues = getInputHistoryValues(
+        values,
+        values.groupHistory as string[],
+        formikBag.props.formRecord.form.groups
+      );
+      const shownElements = filterShownElements(
+        formikBag.props.formRecord.form.elements,
+        values.matchedIds as string[]
+      );
+      return filterValuesByShownElements(inputHistoryValues, shownElements);
+    };
+
     // Needed so the Loader is displayed
     formikBag.setStatus("submitting");
     try {
-      const result = await submitForm(values, formikBag.props.language, formikBag.props.formRecord);
+      const isGroupsCheck = formikBag.props.allowGrouping;
```

Comment:
Need to ensure this matches 

```
const isGroupsCheck = groupsCheck(props.allowGrouping);
```

Which check for allow `allowGrouping` flag but also if the form has groups